### ISSUE TITLE
✏ Create basic ShowCardCarousel component

### ISSUE THREAD

**Issue by Thenlie** [Thenlie/Streamability#60]
Using the ShowCard component, create a horizontally scrollable carousel.

An array of any (reasonable) size movie cards should be able to be sent to the carousel and be rendered on screen. A mockup of this component can be seen below.

The carousel is collapsable 

### Deprecated

![Image](https://user-images.githubusercontent.com/41388783/200219334-b4254296-fd4c-4dc7-9507-a15a4fd335b5.png)

**Comment by Thenlie** [comment]
If we end up creating a placeholder component with functional data but incomplete styles, be sure to create a new ticket to update the styles once the mockup is finished.

**Comment by maxh1231** [comment]
Decided to go with Nuka Carousel.

Docs: https://formidable.com/open-source/nuka-carousel/#nuka-demos
